doc/classes/Control.xml

```diff
@@ -9,7 +9,7 @@
 		[b]Note:[/b] Since both [Node2D] and [Control] inherit from [CanvasItem], they share several concepts from the class such as the [member CanvasItem.z_index] and [member CanvasItem.visible] properties.
 		[b]User Interface nodes and input[/b]
 		Godot propagates input events via viewports. Each [Viewport] is responsible for propagating [InputEvent]s to their child nodes. As the [member SceneTree.root] is a [Window], this already happens automatically for all UI elements in your game.
-		Input events are propagated through the [SceneTree] from the root node to all child nodes by calling [method Node._input]. For UI elements specifically, it makes more sense to override the virtual method [method _gui_input], which filters out unrelated input events, such as by checking z-order, [member mouse_filter], focus, or if the event was inside of the control's bounding box.
+		Input events are propagated through the [SceneTree] from the root node to all child nodes by calling [method Node._input]. For UI elements specifically, it makes more sense to override the virtual method [method _gui_input], which filters out unrelated input events, such as the order of the nodes in the scene tree, [member mouse_filter], focus, or if the event was inside of the control's bounding box. It’s also important to note that properties that control rendering order have no impact on event propagation. This includes properties like z-index, [member CanvasItem.y_sort_enabled], visible behind parent, sort children by y, and being top-level.
```

Comment:
`top_level` does actually affect input event propagation, it blocks it like mouse filter stop.